app/javascript/components/pages/LoggedOutHome.scss

```diff
@@ -0,0 +1,57 @@
+
+
+body {
+    background-color: #00c0d8;
+  }
+  .searchbar{
+    text-align: right;
+  }
+  
+  .footertext{
+    text-align: center;
+  }
+  .cardbody{
+    text-align: center;
+  }
+  hr { display: block; margin-before: 0.5em; margin-after: 0.5em; margin-start: auto; margin-end: auto; overflow: hidden; border-style: inset; border-width: 1px;}
+  
+  
+  
+  .jumbotron {
```

Comment:
Looks like there's some possible unintented over writing of bootstrap/reactstrap styling happening. Best practice is to leave classes from third party libraries alone and then add your own css classes to provide custom styling